src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java

```diff
@@ -16,19 +17,27 @@
 @Configuration
 public class SecurityConfig {
 
+  // default value is user if env variable is not found
+  @Value("${USER_PASSWORD:user}")
+  private String userPassword;
+
+  // default value is admin if env variable is not found
+  @Value("${ADMIN_PASSWORD:admin}")
```

Comment:
same here

Reply:
Now tests ignore getting value of env variables